spec-main/yarn.lock

```diff
@@ -3,283 +3,951 @@
 
 
 "@sinonjs/commons@^1", "@sinonjs/commons@^1.6.0", "@sinonjs/commons@^1.7.0", "@sinonjs/commons@^1.7.2":
-  version "1.8.0"
```

Comment:
There are a lot of changes here, can we maybe limit them to just adding the new package in this PR?

Reply:
yeah, happy to nuke this part entirely!